urllib3/packages/ssl_match_hostname/__init__.py

```diff
@@ -1,5 +1,11 @@
+import sys
+
 try:
-    # Python 3.2+
+    # Our match_hostname function is the same as 3.5's, so we only want to
+    # import the match_hostname function if it's at least that good.
+    if sys.version_info < (3, 5):
```

Comment:
Question here. This bypasses our modifications to `match_hostname` on 3.5 and higher. The two main features that we get from those modifications (as far as I can tell) are 1) ability to handle a missing `ipaddress` module and 2) ability to handle bytestring IP addresses.

Those *should* be non-issues, as >3.5 will never be missing the `ipaddress module` (it's stdlib) and only Python 2 will have bytestrings (Python 3 will always do Unicode strings).

Is that about the shape of it?